specification/support/resource-manager/Microsoft.Support/preview/2019-05-01-preview/support.json

```diff
@@ -1439,7 +1447,7 @@
                 },
                 "contactDetails": {
                     "$ref": "#/definitions/UpdateContactProfile",
-                    "description": "Contact details to be updated on the support ticket."
+                    "description": "Contact details to on the support ticket."
```

Comment:
nit: I thought the previous statement makes more sense and grammatically correct.

Reply:
Good catch, fixed.